packages/app-desktop/jest.setup.js

```diff
@@ -22,6 +22,18 @@ jest.mock('@electron/remote', () => {
 	};
 });
 
+// Mocking bridge().Menu for showSpellCheckerMenu.test.ts
+jest.mock('./services/bridge', () => ({
+	__esModule: true,
+	default: () => ({
+		Menu: {
+			buildFromTemplate: jest.fn().mockReturnValue({
+				popup: jest.fn(),
+			}),
+		},
+	}),
+}));
+
```

Comment:
I don't think a global mock makes sense - that should instead be part of the test itself

Reply:
At first, I added the mock in the `showSpellCheckerMenu.test.ts`, and it broke the app because the test file was part of the `index.ts.` Now that I put it back, the app works!

I will put it back on the test.